build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -193,6 +207,7 @@ Future<void> _createDevCompilerModule(
       '--modules=amd',
       '--no-summarize',
       if (generateFullDill) '--experimental-output-compiled-kernel',
+      if (generateDebugSymbols) '--experimental-output-debug-symbols',
```

Comment:
Should we be retaining the `--experimental` name in the flag we expose from build_web_compilers as well? Usually the names would line up exactly, although I see we didn't do that for the generate-full-dill option...

Reply:
I should probably just make the flags non-experimental in the sdk as well, as we don't have a special way of creating experimental flags, they could be just hidden flags, disabled by default.  My current plan is to just remove them as corresponding feature become mature and are enabled by default.

How does 'output-full-dill' and 'output-debug-symbols' sound for both SDK and build_web_compilers?

I would need to change `output-compiled-kernel`  in a different set of PRs though as changing flags between repos require a handshake between repos:)

Reply:
Changed to `output-debug-symbols` for both SDK and build_web_compilers, will clean up the existing option in a separate PR.

Reply:
Note that changing the flag in the SDK is breaking for old versions of this package so if we are going to change (or remove) those we really should also ensure the old ones still work, or at least don't throw...

Reply:
Sure, I would need a series of changes over a long period of time:)